ext/standard/mail.c

```diff
@@ -74,6 +74,18 @@ static bool php_mail_build_headers_check_field_value(zval *val)
 			}
 			return FAILURE;
 		}
+		/*
+		 * This is a way of writing that violates rfc2822,
+		 * but there are cases where this is accepted.
+		 */
+		if (*(value->val+len) == '\n') {
```

Comment:
it might make sense to do it for `\r` as well.

Reply:
The if block a little above it already does that.

Reply:
Ah ok I see.